OctaneVSPlugin/View/DetailsToolWindowControl.xaml

```diff
@@ -220,53 +362,70 @@
+                                        Value="Visible" />
                             </DataTrigger>
                         </Style.Triggers>
                     </Style>
                 </DockPanel.Style>
 
-                <view:GifImage x:Name="gifImage" Stretch="None" GifSource="/octane-visual-studio-plugin;component/Resources/octane_preloader.gif" AutoStart="True" />
+                <view:GifImage x:Name="gifImage"
+                               Stretch="None"
+                               GifSource="/octane-visual-studio-plugin;component/Resources/octane_preloader.gif"
+                               AutoStart="True" />
 
             </DockPanel>
 
             <DockPanel x:Name="FailedToLoadPanel">
                 <DockPanel.Style>
                     <Style TargetType="DockPanel">
-                        <Setter Property="Visibility" Value="Collapsed" />
+                        <Setter Property="Visibility"
+                                Value="Collapsed" />
                         <Style.Triggers>
-                            <DataTrigger Binding="{Binding Mode}" Value="FailedToLoad">
-                                <Setter Property="Visibility" Value="Visible" />
+                            <DataTrigger Binding="{Binding Mode}"
+                                         Value="FailedToLoad">
+                                <Setter Property="Visibility"
+                                        Value="Visible" />
                             </DataTrigger>
                         </Style.Triggers>
                     </Style>
                 </DockPanel.Style>
 
                 <Grid>
                     <Grid.RowDefinitions>
-                        <RowDefinition Height="50"/>
+                        <RowDefinition Height="50" />
                         <RowDefinition />
                     </Grid.RowDefinitions>
 
-                    <StackPanel Grid.Row="0" HorizontalAlignment="Right" VerticalAlignment="Center" Orientation="Horizontal">
-                        <Button Width="25" Height="25" 
+                    <StackPanel Grid.Row="0"
+                                HorizontalAlignment="Right"
+                                VerticalAlignment="Center"
+                                Orientation="Horizontal">
+                        <Button Width="25"
+                                Height="25"
                                 BorderThickness="0"
                                 Background="{DynamicResource VsBrush.Window}"
                                 Command="{Binding RefreshCommand}"
                                 ToolTip="Refresh details">
                             <!--This text-block show the reload sign as unicode character-->
-                            <TextBlock Text="&#x50;" FontFamily="Wingdings 3"
-                                       FontWeight="Bold" FontSize="24"
+                            <TextBlock Text="&#x50;"
```

Comment:
What does the Text="&#x50;" do? Looks weird

Reply:
Well I didn't touch that part of the code, nevertheless after I investigated what is the meaning of that I have found that the code represents the the reload sign as unicode character.

Reply:
Oh, ok :)